bool get
---

geometry_msgs/PoseArray poses
rospy_tutorials/Floats objectLabels
